usaspending_api/download/delta_models/award_financial_download.py

```diff
@@ -332,7 +334,9 @@ def award_financial_df(spark: SparkSession):
 
 def load_award_financial(spark: SparkSession, destination_database: str, destination_table_name: str) -> None:
     df = award_financial_df(spark)
-    df.write.format("delta").mode("overwrite").saveAsTable(f"{destination_database}.{destination_table_name}")
+    df.write.format("delta").mode("overwrite").partitionBy(
```

Comment:
Similar comment here for `partitionBy()` since the Delta table itself should be partitioned.